app/views/App/Index.html

```diff
@@ -9,7 +9,7 @@ <h1>{{ .Zone }}</h1>
       <p>{{ .rdata }}</p>
     </div>
   </div>
-</header>
+  </header>
```

Comment:
Nit picking but this change is not needed.

Reply:
I agree.